specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-04-01-preview/costmanagement.json

```diff
@@ -0,0 +1,1823 @@
+        "id": {
+          "type": "string",
+          "description": "Scope id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Scope name"
+        },
+        "type": {
+          "type": "string",
+          "description": "Scope type"
+        },
+        "childScope": {
+          "$ref": "#/definitions/Scope"
+        }
+      }
+    },
+    "ViewListResult": {
+      "description": "Result of listing views. It contains a list of available views.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of views.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/View"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "View": {
+      "description": "States and configurations of Cost Analysis.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ViewProperties",
+          "title": "View properties"
+        }
+      }
+    },
+    "ViewProperties": {
+      "type": "object",
+      "description": "The properties of the view.",
+      "properties": {
+        "version": {
+          "description": "View API version used to create the view.",
+          "type": "string"
+        },
+        "queryVersion": {
+          "description": "Query API version to use.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "User input name of the view. Required.",
+          "type": "string"
+        },
+        "scope": {
+          "description": "Cost Management scope to save the view on.",
+          "type": "string"
+        },
+        "createdOn": {
+          "description": "Date the user created this view.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "modifiedOn": {
+          "description": "Date when the user last modified this view.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "query": {
+          "description": "Query body configuration. Required.",
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ReportConfigDefinition"
+        },
+        "chart": {
+          "description": "Chart type of the main view in Cost Analysis. Required.",
+          "type": "string",
+          "enum": [
+            "Area",
+            "Line",
+            "StackedColumn",
+            "GroupedColumn",
+            "Table"
+          ],
+          "x-ms-enum": {
+            "name": "ChartType",
+            "modelAsString": true
+          }
+        },
+        "accumulated": {
```

Comment:
can we make this an enum? bools are typically not recommended. They dont allow for future expansion of states a property can take.

Reply:
Although there are only two states in our scenario, we updated to make it enum to allow further expansion.